src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Models/ApprenticeshipDetailsCsvModel.cs

```diff
@@ -5,20 +5,20 @@
 
 namespace SFA.DAS.ProviderCommitments.Web.Models
 {
-    public class ApprenticeshipDetailsCsvViewModel 
+    public class ApprenticeshipDetailsCsvModel 
     {
-        public static implicit operator ApprenticeshipDetailsCsvViewModel(ApprenticeshipDetails model)
+        public static implicit operator ApprenticeshipDetailsCsvModel(ApprenticeshipDetails model)
         {
-            return new ApprenticeshipDetailsCsvViewModel
+            return new ApprenticeshipDetailsCsvModel
             {
                 ApprenticeName = $"{model.ApprenticeFirstName} {model.ApprenticeLastName}",
                 Uln = model.Uln,
                 Employer = model.EmployerName,
                 CourseName = model.CourseName,
-                PlannedStartDate = model.PlannedStartDate,
-                PlannedEndDate = model.PlannedEndDateTime,
+                PlannedStartDate = model.PlannedStartDate.ToString("MMM yyyy"),
```

Comment:
Theres a ToGdsDate i think that we should use instead for this

Reply:
@chrisfoster76 would be good to get your input on this - should we omit the day from the CSV download? Is the start date always the first of the month?